test/line-endings/CR.js

```diff
@@ -1 +1 @@
-// > 2 * 3 * 7
// 42
+// > 2 * 3 * 7
// 42

```

Comment:
This reverts a change unintentionally introduced in <https://github.com/davidchambers/doctest/commit/03e285252e12c56b4cab7763199f9fe9992428fa>.